jwst/ami/oifits.py

```diff
@@ -81,37 +85,36 @@ def make_obsarrays(self):
             self.fringe_amplitudes[i, :] = nrmslc.fringeamp
             self.closure_phases[i, :] = nrmslc.redundant_cps  # CPs in radians
             self.t3_amplitudes[i, :] = nrmslc.t3_amplitudes
-            self.q4_phases[i, :] = nrmslc.q4_phases # quad phases in radians
+            self.q4_phases[i, :] = nrmslc.q4_phases  # quad phases in radians
             self.closure_amplitudes[i, :] = nrmslc.redundant_cas
-            self.pistons[i, :] = nrmslc.fringepistons # segment pistons in radians
+            self.pistons[i, :] = nrmslc.fringepistons  # segment pistons in radians
             self.solns[i, :] = nrmslc.soln
 
-        self.fringe_amplitudes_squared = self.fringe_amplitudes ** 2  # squared visibilities
+        self.fringe_amplitudes_squared = (
+            self.fringe_amplitudes**2
```

Comment:
This example makes me think the past comment had a peculiar interaction with the parentheses. Maybe nothing to be done with operator spacing after all.

Reply:
Sorry if I missed it, what was the previous comment?

Reply:
Sorry, regarding this comment/change: https://github.com/spacetelescope/jwst/pull/9081#discussion_r1927687117

Reply:
Yes it was caused by the fact that the original code had parentheses around the exponent, which made the formatter think it was a complex operation and should be spaced.  Without the parentheses, the formatter thinks `10**-3` is best